README.md

```diff
@@ -43,69 +43,95 @@ remotes::install_github("mrc-ide/frogger", upgrade = FALSE)
-| incrr_age          | incidence_relative_risk_age | Relative risk of incidence by age                                                            |
-| cd4_initdist       | ?                           | Distribution of infections by cd4 category upon infection                                    |
-| cd4_prog           | cd4_progression             |                                                                                              |
-| cd4_mort           | cd4_mortality               |                                                                                              |
-| art_mort           | art_mortality               |                                                                                              |
-| artmx_timerr       | ?                           |                                                                                              |
-| art15plus_num      | ?                           |                                                                                              |
-| art15plus_isperc   | ?                           |                                                                                              |
-| artcd4elig_idx     | ?                           |                                                                                              |
-| art_alloc_method   | ?                           |                                                                                              |
-| art_alloc_mxweight | ?                           |                                                                                              |
-| scale_cd4_mort     | scale_cd4_mortality         |                                                                                              |
-| art_dropout        | ?                           |                                                                                              |
+In 4 structs as part of the `Parameters`, named `Demography`, `Incidence`, `NaturalHistory` and `Art`
+
+| Leapfrog           | Frogger                                  | Details                                                                                                                                                               |
+|--------------------|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| basepop            | demography.base_pop                      | Population data by age group and sex                                                                                                                                  |
+| sx                 | demography.survival                      | Probability of surviving between ages, from 0 to 1, 1 to 2, ..., 79 to 80+ and 80+ to 80+                                                                             |
```

Comment:
It would be nice for this to refer to survival _probability_, and also to age somehow. In demographic conventions the _x_ refers to age. So this term in demographic convention would be written in notation as _S<sub>x</sub>_, denoting "the probability of [S]urviving from age [x] to x+1".

(Not suggesting we retain that; just for background info on what `sx` means)

Reply:
`demography.survival_probability`?